### PR TITLE
ref(ui): Remove platformicon icon font. Bump platformicons to 3.0.0

### PR DESCRIPTION
**Pull request by ckj**
This gets rid of the old platformicons icon font and replaces it, as well as existing svg-based-based platformicons, with @doralchan's new set found in Platformicons 3.

**Before:**

Screen Shot 2020-09-09 at 4 43 06 PM

**After:**

Screen Shot 2020-09-09 at 4 40 49 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectInstall/createProject.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ class CreateProject extends React.Component<Props, State> {
         <div>
           <FormLabel>{t('Project name')}</FormLabel>
           <ProjectNameInput theme={theme}>
-            <ProjectPlatformIcon monoTone platform={platform} />
+            <StyledPlatformIcon platform={platform} />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/createProject.tsx:95]: Should we pass `size=20px` here? The java icon looks wrong here:

image

Comment (doralchan) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.tsx:95]: Hmm I can try updating it in the next batch to have a higher contrast? I have new logos to add for serverless and connect anyways

Comment (ckj) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.tsx:95]: We should do something, as we're showing these at 16px in at least one pretty prominent spot (the project selector in the global header). Maybe we should just enforce a min-size of 20px everywhere?

Comment (doralchan) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.tsx:95]: That's definitely affect a lot of locations. Here's what it looks like with a new treatment:

**Before and after (2 color versions with stroke width applied)**
Screen Shot 2020-09-10 at 3 24 08 PM

Comment (ckj) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.tsx:95]: @billyvg we're going to adjust the icon itself to read better at smaller sizes.

**Review by billyvg — APPROVED** [review]
I may try to see if we can codesplit these as this adds 1% to our bundle

**Pull request merged**